test/ui/cypress/integration/serving.spec.js

```diff
@@ -91,7 +91,10 @@ describe('OCP UI for Serverless', () => {
 
   const showcaseKsvc = new ShowcaseKservice()
 
-  it('can deploy kservice and scale it', () => {
+  it('can deploy kservice and scale it',
+    // TODO: Remove the increased timeout when https://issues.redhat.com/browse/ODC-5685 is fixed.
+    { defaultCommandTimeout: 300000 },
```

Comment:
Maybe we can move it to a specific invocation that fails?

Reply:
Pushed.